backend/src/main/kotlin/org/loculus/backend/model/SubmitModel.kt

```diff
@@ -344,14 +353,17 @@ class SubmitModel(
 
         if (metadataKeysNotInSequences.isNotEmpty() || sequenceKeysNotInMetadata.isNotEmpty()) {
             val metadataNotPresentErrorText = if (metadataKeysNotInSequences.isNotEmpty()) {
-                "Metadata file contains ${metadataKeysNotInSequences.size} ids that are not present " +
-                    "in the sequence file: " + metadataKeysNotInSequences.toList().joinToString(limit = 10) + "; "
+                "Metadata file contains ${metadataKeysNotInSequences.size} FASTA ids that are not present " +
+                    "in the sequence file: " + metadataKeysNotInSequences.toList().joinToString(limit = 10) {
+                        "'$it'"
+                    }
```

Comment:
```suggestion
                    "in the sequence file: " + metadataKeysNotInSequences.toList().joinToString(limit = 10) {
                        "'$it'"
                    } + ". "
```
to add a separate for potential next messages.